lib/replace-selection-items.js

```diff
@@ -0,0 +1,14 @@
+import {splice} from 'immutable-array-methods';
+import getSelectionItems from './get-selection-items';
+
+export default (articleJson, fn) => {
+  const items = getSelectionItems(articleJson);
+
+  if (!items.length) {
+    return articleJson;
+  }
```

Comment:
I'm pretty sure we handle this in `immutable-array-methods` - that doing splice w deleteCount === 0 & no items will be fine.